dataplane/unittest/vrf_lpm_linear.cpp

```diff
@@ -0,0 +1,83 @@
+#include <gtest/gtest.h>
+
+#include "../vrf.h"
+#include "random_prefixes.h"
+#include "vrf_lpm_common.h"
+#include "vrf_lpm_map.h"
+
+namespace dataplane::vrflpm
+{
+
+static constexpr size_t size_of_mem = 1024 * 1024 * 100;
+
+TEST(VrfLpm4, SimpleMap)
+{
+	VrfLpmMap<ipv4_prefix_t, uint32_t, uint32_t> vrf_map;
+	SimpleTestVrfLpm4(vrf_map);
+}
+
+TEST(VrfLpm4, SimpleLinear)
+{
+	std::unique_ptr<uint8_t[]> buffer = std::make_unique<uint8_t[]>(size_of_mem);
+	VrfLpm4Linear::BlocksAllocator allocator;
+	allocator.Init(buffer.get(), size_of_mem, YANET_RIB_VRF_MAX_NUMBER);
+	VrfLpm4Linear vrf_linear(allocator);
+
+	SimpleTestVrfLpm4(vrf_linear);
+}
```

Comment:
We could **greatly** reduce code duplication by utilizing GoogleTests fixtures and `TYPED_TEST_SUITE` to parametrize tests for both `VrfLpmMap` and `VrfLpmLinear`